src/main/java/com/github/javafaker/Faker.java

```diff
@@ -291,7 +291,12 @@ public Cat cat() {
      * @param key
      * @return
      */
-    public String resolve(String key) {
+    @Override
+    public String resolve(String key){
+        return resolve(key, null);
+    }
+
+    public String resolve(String key, Object... args) {
```

Comment:
What's the use case for this?  Resolve basically resolves to child objects of Faker.  Why not just call the objects themselves?